app/charts/area/areas-state.tsx

```diff
@@ -51,6 +52,8 @@ import { sortByIndex } from "@/utils/array";
 import { estimateTextWidth } from "@/utils/estimate-text-width";
 import { makeOrdinalDimensionSorter } from "@/utils/sorting-values";
 
+import useChartFormatters from "../shared/use-chart-formatters";
```

Comment:
That's totally a huge nit, but I think it could be "@/charts/shared/...", as like with the imports above?